examples/restricted-assets/README.md

```diff
@@ -0,0 +1,48 @@
+# Asset Transfer using custom logic
+
+This example demonstrates authorizing asset transfer transactions based on custom logic by smart contract.
+
+## Overview
+
+Using Algorand Standard Assets, developers can create a token that represents either a real world or digital good, service, or resource in a matter of minutes. No smart contract is required. These assets are very powerful and allow for flexibility in how they are traded and controlled. Typically assets are traded as any other token on the chain using `asset transfer transaction`.
+
+This is fine for most cases, but what if you want some custom logic to execute to approve the transfer. This may be the case when you have one of the following scenarios.
```

Comment:
```suggestion
This is fine for most cases, but what if you want some custom logic to execute to approve the transfer? This may be the case when you have one of the following scenarios:
```